zilencer/migrations/0027_backfill_remote_realmauditlog_extradata_to_json_field.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 4.0.7 on 2022-09-30 20:30
+
+import ast
+from typing import Callable, Type
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import JSONField, Model
+from django.db.models.functions import Cast
+
+BATCH_SIZE = 5000
+
+
+def do_bulk_backfill_extra_data(
```

Comment:
Do we want to put a comment somewhere in here that this code is mostly-common with `zerver/migrations/0424_backfill_remote_realmauditlog_extradata_to_json_field.py` (and vice versa) so we update both migrations at the same time, if we have to fix something in the future?